lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -183,34 +183,53 @@ const Reportback = {
    * @param  {array} data  Array of Kudos objects from API.
    * @return {array}
    */
-  getReactions: function(data) {
-    let reactions = [];
+  getReactions: function(data, reactionFromRogue) {
+    if (reactionFromRogue) {
+      var reactions = [];
 
-    forEach(this.reactions.approved, (term) => {
-      const termId = this.reactions.terms[term];
-
-      let reaction = {
+      var reaction = {
         kudosId: null,
-        term: term,
-        termId: termId,
+        term: null,
+        termId: null,
         total: 0,
         userReacted: false,
       };
 
-      if (data.length) {
-        forEach(data, (record) => {
-          if (record.term.name === term) {
-            reaction.kudosId = record.current_user.kudos_id;
-            reaction.term = record.term.name;
-            reaction.termId = record.term.id;
-            reaction.total = record.term.total;
-            reaction.userReacted = record.current_user.reacted;
-          }
-        });
-      }
+      reaction.term = 'heart';
+      reaction.termId = 641,
+      reaction.total = data.total;
+      reaction.userReacted = data.liked_by_current_user;
 
       reactions.push(reaction);
-    });
+    } else {
+      let reactions = [];
+
+      forEach(this.reactions.approved, (term) => {
+        const termId = this.reactions.terms[term];
+
+        let reaction = {
+          kudosId: null,
```

Comment:
I'd recommend trying to do most of this work (formatting the data into `kudosId`, `term`, `termId`, `total` and `userReacted`) within Rogue, so you don't have to then transform it again within the Phoenix client-side code. That way Phoenix doesn't have to be aware of any of the implementation details of the new setup.